ui/pom.xml

```diff
@@ -31,6 +31,13 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
+            <groupId>org.jboss.windup.graph</groupId>
+            <artifactId>windup-graph-impl</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency> 
+
+        <dependency>
```

Comment:
Additionally, we can clean up the API on reloading later, but I'm sure there is a better way to do that, than with a boolean.